lib/web/apiserver.go

```diff
@@ -243,6 +243,12 @@ type Config struct {
 	// HealthCheckAppServer is a function that checks if the proxy can handle
 	// application requests.
 	HealthCheckAppServer healthCheckAppServerFunc
+
+	UiConfig UiConfig
+}
+
+type UiConfig struct {
```

Comment:
I think you've defined this struct 4 or 5 times by now.

I know our config package does tend to get a bit repetitive, but is there any way we can reduce at least a couple of these?

Reply:
Trimmed it down to 1 definition. Didn't know I could combine json/yaml field tags together and that was very helpful